modules/admin_manual/pages/installation/letsencrypt/nginx.adoc

```diff
@@ -1,27 +1,50 @@
 = Setup NGINX
 :toc: right
 :toclevels: 1
+:diffie-hellman-url: https://en.wikipedia.org/wiki/Diffie–Hellman_key_exchange
+:forward-secrecy-url: https://scotthelme.co.uk/perfect-forward-secrecy/
+:ssl_dhparam: http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_dhparam
```

Comment:
please use the `-url` suffix in this and the following attribute, to keep consistency with the de facto standard that we have in place.